prow/pod-utils/downwardapi/jobspec_test.go

```diff
@@ -37,14 +44,13 @@ func TestEnvironmentForSpec(t *testing.T) {
 				BuildID:   "0",
 				ProwJobID: "prowjob",
 			},
-			expected: map[string]string{
+			expected: decorateWithGitEnvs(map[string]string{
```

Comment:
In general I'm not sure that "DRY" applies to tests - we try to keep each test case self-contained, so that failure domains are tiny and you can tell when you broke something specific, or not. Would it be hard to just enumerate these env vars everywhere, like we are already doing for `$CI="true"` ?

Reply:
Nope, I can do that. I just thought it was kind of silly, but I do not have any objections.